src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+import createBook from '../../services/bookService';
+
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
```

Comment:
Kindly make sure that the value of each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like: `const ADD_BOOK = bookstore/books/ADD_BOOK` and `const REMOVE_BOOK = bookstore/books/REMOVE_BOOK`